pkgs/jovian-stubs/dmidecode

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+>&2 echo "[JOVIAN] $0: stub called with: $*"
+
+if [ "$*" == "-t 11" ]; then
+    >&2 echo "[JOVIAN] $0: replying with cached data"
+    cat /run/jovian/dmidecode-11.txt
+    exit 0
+else
+    >&2 echo "[JOVIAN] $0: don't know how to handle $*"
+fi
```

Comment:
[So neither of us forget]

```suggestion
fi
exit 126
```

Or any other code you deem appropriate.

(Why 126? [I have exit codes, everyone does, and I arbitrarily looked at this table and guessed](https://tldp.org/LDP/abs/html/exitcodes.html))